Sources/MapboxCoreNavigation/ReplayLocationManager.swift

```diff
@@ -18,9 +18,10 @@ open class ReplayLocationManager: NavigationLocationManager {
     /**
      `locations` to be replayed.
      */
-    public var locations: [CLLocation]! {
+    public var locations: [CLLocation] {
```

Comment:
Technically it's a breaking change in public API 🙂
It was possible to write:
```
if manager.locations != nil {
```

Reply:
Thanks, will mention this in the changelog 👍